### PR TITLE
correction on GPS output

### PR DESCRIPTION
**Pull request by qiangRadioss**
#### Description of the feature or the bug

crash results of solid model using void material in case of GPS (or GPSTRAIN) output 

#### Description of the changes

Corrections have been done for GPS or GPSTRAIN output to avoid Engine crash or wrong results

### CODE REVIEW

File: engine/source/output/anim/generate/tensor6.F [hunk 1]
```diff
@@ -4359,7 +4360,38 @@ SUBROUTINE TENSGPS3(ELBUF_TAB,FUNC1   ,FUNC2   ,IPARG   ,GEO     ,
...
+               ELSE
+                 GAMA(1)=ONE
+                 GAMA(2)=ZERO
+                 GAMA(3)=ZERO
+                 GAMA(4)=ZERO
+                 GAMA(5)=ONE
+                 GAMA(6)=ZERO
+               END IF
+             END IF
+                 N1 = ONE
```
Comment (laurent-altr) [inline engine/source/output/anim/generate/tensor6.F:4379]: The indentation does not seem good. Could you review it?

**Review by laurent-altr — CHANGES_REQUESTED** [review]

File: engine/source/output/anim/generate/tensor6.F (hunk 1 above)
Comment (qiangRadioss) [reply engine/source/output/anim/generate/tensor6.F]: done.
thanks.

File: engine/source/output/anim/generate/tensgpstrain.F [hunk 2]
```diff
@@ -254,7 +255,45 @@ SUBROUTINE TENSGPSTRAIN(ELBUF_TAB,FUNC1   ,FUNC2   ,IPARG   ,GEO     ,
...
+               ELSE
+                 GAMA(1)=ONE
+                 GAMA(2)=ZERO
+                 GAMA(3)=ZERO
+                 GAMA(4)=ZERO
+                 GAMA(5)=ONE
+                 GAMA(6)=ZERO
+               END IF
+             END IF
+                   N1 = FOURTH
```
Comment (laurent-altr) [inline engine/source/output/anim/generate/tensgpstrain.F:279]: Indentation

File: engine/source/output/anim/generate/tensgpstrain.F [hunk 3]
```diff
@@ -897,11 +899,50 @@ SUBROUTINE GPSSTRAIN_SKIN(ELBUF_TAB,FUNC1   ,FUNC2   ,IPARG   ,
...
```
Comment (laurent-altr) [inline engine/source/output/anim/generate/tensgpstrain.F:924]: indentation

**Review by laurent-altr — CHANGES_REQUESTED** [review]
@qiangRadioss, sorry I wasn't clear in my previous review: I would like all the indentation issues to be fixed, but  I wrote a comment only on one of them.

Please note that your IDE should have automatic indentation fixing capabilities.

File: engine/source/output/anim/generate/tensgpstrain.F (hunk 3 above)
Comment (qiangRadioss) [reply engine/source/output/anim/generate/tensgpstrain.F:924]: done

**Pull request closed without merging**